cmd/launcher/extension.go

```diff
@@ -209,6 +209,9 @@ func commonRunnerOptions(logger log.Logger, k types.Knapsack) []runtime.OsqueryI
 		kolidelog.WithKeyValue("osqlevel", "stdout"),
 	)
 
+	// Only enable watchdog internally for now
+	enableWatchdog := k.UpdateChannel() == "beta" || k.UpdateChannel() == "nightly"
```

Comment:
```suggestion
	enableWatchdog := k.UpdateChannel() == "nightly"
```

I feel like we should test this somewhat first.

Reply:
I added some testing notes to the PR description